addons/api/fnc_setRevealToAI.sqf

```diff
@@ -19,6 +20,8 @@ INFO_2("%1 called with: %2",QFUNC(setRevealToAI),_this);
 
 params ["_var"];
 
+if (!hasInterface) exitWith {false};
```

Comment:
Sounds good then, add that to coding guidelines? :) (something along the lines of "in case of API calls on wrong machines, silently exit")